//go:build !try

// Code generated by github.com/goghcrow/go-try DO NOT EDIT.
package test

import (
	"fmt"
)

func named_error() (a int, err error) {
	_, 𝗲𝗿𝗿𝟭 := ret1Err[bool]()
	if 𝗲𝗿𝗿𝟭 != nil {
		err = 𝗲𝗿𝗿𝟭
		return
	}

	a = 42
	return
}
func catch_and_log() (a int, err error) {
	defer func() {
		if err != nil {
			fmt.Println(err)
		}
	}()
	_, 𝗲𝗿𝗿𝟭 := ret1Err[bool]()
	if 𝗲𝗿𝗿𝟭 != nil {
		err = 𝗲𝗿𝗿𝟭
		return
	}

	a = 42
	return
}
